tests/integration/src/test/scala/mesosphere/marathon/integration/SharedMemoryIntegrationTest.scala

```diff
@@ -0,0 +1,232 @@
+        ))
+      )),
+      containers = Seq(
+        MesosContainer(
+          name = "task1",
+          exec = Some(raml.MesosExec(raml.ShellCommand(appMockCommand("$ENDPOINT_TASK1")))),
+          resources = raml.Resources(cpus = 0.1, mem = 32.0),
+          endpoints = Seq(raml.Endpoint(name = "task1", hostPort = Some(0))),
+          healthCheck = Some(MesosHttpHealthCheck(portIndex = Some(PortReference("task1")), path = Some("/ping"))),
+          volumeMounts = Seq(
+            VolumeMount(Some("python"), s"$containerDir/python", true)
+          ),
+          linuxInfo = Some(state.LinuxInfo(
+            seccomp = None,
+            ipcInfo = Some(state.IPCInfo(
+              ipcMode = state.IpcMode.ShareParent,
+              shmSize = None
+            ))
+          ))
+        ),
+        MesosContainer(
+          name = "task2",
+          exec = Some(raml.MesosExec(raml.ShellCommand(appMockCommand("$ENDPOINT_TASK2")))),
+          resources = raml.Resources(cpus = 0.1, mem = 32.0),
+          endpoints = Seq(raml.Endpoint(name = "task2", hostPort = Some(0))),
+          healthCheck = Some(MesosHttpHealthCheck(portIndex = Some(PortReference("task2")), path = Some("/ping"))),
+          volumeMounts = Seq(
+            VolumeMount(Some("python"), s"$containerDir/python", true)
+          ),
+          linuxInfo = Some(state.LinuxInfo(
+            seccomp = None,
+            ipcInfo = Some(state.IPCInfo(
+              ipcMode = state.IpcMode.ShareParent,
+              shmSize = None
+            ))
+          ))
+        )
+      ),
+      volumes = Seq(
+        HostVolume(Some("python"), s"$projectDir/src/test/resources/python")
+      ),
+      unreachableStrategy = state.UnreachableDisabled,
+      upgradeStrategy = state.UpgradeStrategy(0.0, 0.0),
+      networks = Seq(HostNetwork),
+      instances = 1,
+      role = BaseMarathon.defaultRole
+    )
+
+    val body:Pod = Raml.toRaml(pod)
+    val bodyString = Json.prettyPrint(Pod.playJsonFormat.writes(body))
+
+    When("The pod is deployed")
+    val createResult = marathon.createPodV2(pod)
+    createResult should be(Created)
+    waitForDeployment(createResult)
+
+    val (ipcInfo1:String, ipcInfo2:String) = eventually {
+      marathon.status(pod.id) should be(Stable)
+      val status = marathon.status(pod.id).value
+
+      val hosts = status.instances.flatMap(_.agentHostname)
+      hosts should have size (1)
+      val ports = status.instances.flatMap(_.containers.flatMap(_.endpoints.flatMap(_.allocatedHostPort)))
+      ports should have size (2)
+      val facade1 = AppMockFacade(hosts.head, ports.head)
+      val facade2 = AppMockFacade(hosts.head, ports.last)
+
+      val ipcInfoString1 = facade1.get(s"/ipcinfo").futureValue
+      val ipcInfoString2 = facade2.get(s"/ipcinfo").futureValue
+
+      logger.info("IPCInfo1 is: " + ipcInfoString1)
+      logger.info("IPCInfo2 is: " + ipcInfoString2)
+
+      ipcInfoString1 should include("=======")
```

Comment:
This is a really odd assertion :-)

Reply:
Yeah... Good that we have reviews, i was so done with this PR. Was glad that it did work somehow ;) I've cleaned that up